app/assets/javascripts/oxalis/controller/scene_controller.js

```diff
@@ -335,8 +352,13 @@ class SceneController {
       this.setClippingDistance(clippingDistance);
       this.setDisplayCrosshair(displayCrosshair);
       this.setDisplayPlanes(tdViewDisplayPlanes);
-      this.setInterpolation(Store.getState().datasetConfiguration.interpolation);
     });
+    listenToStoreProperty(
+      storeState => storeState.datasetConfiguration.interpolation,
+      // Setting interpolation is expensive as THREE will re-render everything and also re-upload
```

Comment:
Right, before my changes this was always done when the store changed, which is expensive. Now, I made the listener more concrete. However, the comment is obsolete anyway, since the interpolation is done in the shader now. I'll remove the comment.